### DISCUSSION TITLE
[2021] - Summer Roadmap (UX/UI/Frontend)

### DISCUSSION THREAD

**Discussion by TBouder** [fameal/ape-tax#81] [Roadmaps]
## What 🧭
We had some discussions on the nice-to-have features on ape.tax and agreed on a set of features which would be nice to implement to improve the user experience, the strategist way to add strategies and the overall dAPP flow.  
Nothing is written in stone, some suggestions could be canceled, reverted or new ones could be added. Anything here can be discussed.

## Hotlist 🔥
- [X] Add a script to be able to ease the deployment of a new vault in ape.tax (#78)
- [X] Align the logos between all the lines on the homePage to avoid gap and visual 'error' (#80)
- [X] Add a `new` tag to indicate the new vaults to users (#79)
- [X] Hide all the `withdraw` vaults. Only display them if you need to withdraw funds from them (#83)
- [x] Clean the usage of drizzle (avoid full-dapp reloading for instance) (#90)
- [x] Display the APY for the vaults (#87)
- [x] Add a `return home` button on the vault detail page (#89)
- [x] Add multiple wallet support (#90)
- [x] Avoid useless page reload (#90)
- [x] Migration to React (#90)
- [x] Add a withdraw button to only withdraw some shares of the vault (#93)
- [X] Fetch the prices from coingecko vault-wise (#91)
- [x] Fetch the Yearn data app-wise instead of on each vault page
- [x] Add some nice animations for the loaders

## Maybelist 🌱
- [ ] Your idea!

**Comment by saltyfacu** [comment]
It'd be nice to have something like this as a twitter image when sharing a vault URL. The example is: https://ape.tax/yfi-wifey
 
<img width="221" alt="Screenshot 2021-08-23 at 10 40 55" src="https://user-images.githubusercontent.com/839844/130417417-ac7d2568-7cea-410e-86fe-48998d6e12ae.png">
